plugin/src/main/java/com/playmonumenta/scriptedquests/zones/ZoneManager.java

```diff
@@ -212,11 +212,9 @@ public boolean hasProperty(Player player, String layerName, String propertyName)
 	 */
 	public void reload(Plugin plugin, CommandSender sender) {
 		mQueuedReloadRequesters.add(sender);
-		if (mAsyncReloadHandler != null) {
-			if (sender != null) {
-				sender.sendMessage(ChatColor.GOLD + "Zones already reloading, and will run again when done.");
-			}
-		} else {
+
+		sender.sendMessage(ChatColor.GOLD + "Zone reload started in the background, you will be notified of progress.");
```

Comment:
If the sender is the console (at least for server startup), this will throw a NPE.

Reply:
I'm also not sure the message is needed if mAsyncReloadHandler is null, because that indicates it will start right away.

Reply:
Ah, missed that you removed the message for when your entry in the queue starts. Nevermind on the second part.

Reply:
Good catch.